src/OpenTelemetry.Exporter.Prometheus/PrometheusExporterOptions.cs

```diff
@@ -48,7 +49,23 @@ public class PrometheusExporterOptions
         /// Gets or sets the prefixes to use for the http listener. Default
         /// value: http://*:80/.
         /// </summary>
-        public IReadOnlyCollection<string> HttpListenerPrefixes { get; set; } = new string[] { "http://*:80/" };
+        public IReadOnlyCollection<string> HttpListenerPrefixes
+        {
+            get => this.httpListenerPrefixes;
+            set
+            {
+                foreach (string inputUri in value)
+                {
+                    if (!(Uri.TryCreate(inputUri, UriKind.Absolute, out var uri) &&
+                        (uri.Scheme == Uri.UriSchemeHttp || uri.Scheme == Uri.UriSchemeHttps)))
+                    {
+                        throw new ArgumentException("Prometheus server path should be a valid uri with http/https protocol.");
```

Comment:
```suggestion
                        throw new ArgumentException("Prometheus server path should be a valid URI with http/https scheme.", nameof(HttpListenerPrefixes));
```

Reply:
I don't know if "Prometheus server path" is a good name, and I cannot think of a better one at this moment.

Reply:
I know! I struggled with the name as well and apparently this is the best I could come up with so far 😅 .

Reply:
Maybe I should update the http/https to be upper case as well?

Reply:
I think http/https is fine, and probably better in this case.

https://datatracker.ietf.org/doc/html/rfc3986#section-3.1

![image](https://user-images.githubusercontent.com/17327289/148617347-f3b7fb94-70bb-4559-844b-d31209f6137f.png)

Reply:
> I know! I struggled with the name as well and apparently this is the best I could come up with so far 😅 .

Finding good names is probably the hardest thing in coding #2371.